### ISSUE TITLE
Expand About Page

### ISSUE THREAD

**Issue by odysseywestra** [mypaint/mypaint.github.io#7]
I would like to expand the About page to give a brief history of MyPaint. And also a place for developers and contributors to introduce themselves. An example of contributor info would be like how http://freedompenguin.com/the-contributors/ ,which I contribute, to dose it. So in all, I would need a brief introduction and where you contribute of @jonnor , @martinxyz , @achadwick, myself and, who ever else contributes to MyPaint as well on a regular basis.

**Comment by martinxyz** [comment]
For early history there were two posts from me in the wordpress blog which I've copied to my personal page now: http://log2.ch/tag/mypaint/ - feel free to use/edit/copy/link/discard as you see fit.

**Comment by achadwick** [comment]
Motivation and goals pages too, scraped: https://gist.github.com/achadwick/61f8c9f4f820ebdf5495

**Comment by odysseywestra** [comment]
Okay I just expanded the http://mypaint.org/about to give a brief history of how MyPaint came to be. Let me know what your guys thoughts are and what else I should add to the about page before closing this issue.

**Comment by achadwick** [comment]
I think the copy could be punchier, and there's a few minor typos and grammar blips. If it's OK, I'd like change it around a bit. I'll send a PR, let me know what you think.

I've been updating the wiki recently, and as part of researching [a new page](https://github.com/mypaint/mypaint/wiki/Writing-Documentation), I stumbled across **<http://www.hemingwayapp.com/>** ([h/t Charlotte Spencer](https://the-pastry-box-project.net/charlotte-spencer/2015-september-16)). It's a great little tool for making terse punchy copy.

**Comment by odysseywestra** [comment]
Alright Anything left we need to add to the About Page. Maybe some of the pictures of older version of MyPaint?

**Comment by achadwick** [comment]
The page is basically a timeline-history. Adding screenshots or stock-imagey points of interest could help break up the wall of text. It's something whose graphical style may need to be fit into the site as a while (#4), for for now, but we could start with rectangular thumbs and floats

![staggered](https://cloud.githubusercontent.com/assets/61299/11607830/7fcc6e06-9b4e-11e5-8434-9c23225f3edc.jpg)

and move to something fancier and more linear later, maybe the vertically centred

![linear](https://cloud.githubusercontent.com/assets/61299/11607834/a596cdca-9b4e-11e5-9610-cc35d044076e.jpg)

More whitespace is nice! ☺

I think @martinxyz has a stash of ancient screenshots somewhere? There's a screenshot of [v0.2](https://github.com/mypaint/mypaint/releases/tag/v0.2) in the releases timeline on github that could be incorporated with some nice framing/thumbnailing/vignetting.

**Comment by achadwick** [comment]
Perhaps starting with the history might not be so great? Apply a bit of [inverted pyramid](https://en.wikipedia.org/wiki/Inverted_pyramid) goodness

* Terse summary paragraph saying what MyPaint *actually is right now*, then introducing the timeline
* Two history paras, no more.
* Longer description of the current state, starting with the end of the timeline.
  * It should have some visual element to indicate that this is how it is now.
  * Links to a new screenshots page, or a little gallery strip?
* End with the download button in case the reader is sold this far down the page!

**Comment by odysseywestra** [comment]
Yeah, my plan right now is get all the information that we need. Then we'll adjust the information to according site theme we use. That way we prevent breaking the site, but I will take note of it in #4.

I do have screenshots from the theme backup that @deevad had, but I do not know which screenshot goes to which version.

**Comment by odysseywestra** [comment]
Good suggestions though @achadwick. ^^

**Comment by Deevad** [comment]
If you need old screenshot, I have this one from March 2009, using Mypaint 0.6 on Linux Mint 6 Felicia :-)
![2009-03-mypaint-06-v1](https://cloud.githubusercontent.com/assets/4021867/11611404/53c99224-9bcc-11e5-8df8-6d2949492cf3.jpg)

**Comment by odysseywestra** [comment]
Perfect that exact one we would need since V0.6 is mentioned in the about
page.

**Comment by achadwick** [comment]
@odysseywestra Okay, NP. I think the text for the about page is basically all there, and if page layout is going to be revisited later, this one could be closed now.

**Comment by odysseywestra** [comment]
Okay put the put the image on the about page and fix an annoying image resizing issue with some css as well. So as of  2db7065 the about page is complete information wise.

**Comment by martinxyz** [comment]
I agree with @achadwick , my expectation of an "about" page is to talk first of all about the program and the context in which it stands today. And if it is meant as a landing page, visitors will be like "I have used paint.net and read about this GIMP thing, should I try MyPaint? Who is it for?" -- They are in a hurry and at this point and skimming over the first two sentences. At this point, the names of any developers or the date when the project came alive are among the last things they want to know. (And judging by the spam I'm getting, they also want to know that we're not selling paint nor hand-crafted copies of classical paintings.)

I don't really have a secret stash of old mypaint screenshots, except maybe those: http://log2.ch/old/old-mypaint/images-recover/ I have also found a very old dump (2009-06) of the wordpress FTP if that's any help (I guess not).
